x-pack/legacy/plugins/siem/public/components/page/overview/overview_host_stats/index.tsx

```diff
@@ -25,6 +25,7 @@ interface OverviewHostProps {
   loading: boolean;
 }
 
+// eslint-disable-next-line complexity
```

Comment:
This function appears to have a repeating pattern that can be refactored, but for now the new entries follow the established (declarative) pattern.